qiskit_quantuminspire/qi_backend.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, List, Union
+
+from compute_api_client import BackendType, Metadata
+from qiskit.circuit import QuantumCircuit
+from qiskit.providers import BackendV2 as Backend
+from qiskit.providers.options import Options
+from qiskit.transpiler import Target
+
+from qiskit_quantuminspire.qi_jobs import QIJob
+
+
+class QIBackend(Backend):  # type: ignore[misc]
```

Comment:
Why the type ignore? (In QI job as well).

Reply:
Mypy is unable to fetch type info of third party library (Qiskit in this case), so it deduces type of Backend as Any resulting in the following error.  
 `error:` Class cannot subclass "Backend" (has type "Any")  [misc]`
 
 [This](https://stackoverflow.com/questions/49888155/class-cannot-subclass-qobject-has-type-any-using-mypy) stack-overflow thread suggests to silence this specific error

Reply:
added comments